mmhuman3d/core/cameras/camera_parameters.py

```diff
@@ -448,9 +448,13 @@ def check_item(self, key: Any, val: Any) -> None:
                 CameraParameter.SUPPORTED_KEYS.
             TypeError:
                 Value's type doesn't match definition.
+        Returns:
+            key (Any): The input key.
+            val (Any): The value casted into correct format.
         """
         self.__check_key__(key)
-        self.__check_value_type__(key, val)
+        formatted_val = self.__check_value_type__(key, val)
```

Comment:
@Maoxie suggests to replace `check` with `validate`

Reply:
Fixed in c3edd34.